Quick-Start/Prerequisite.md

```diff
@@ -1,101 +1,101 @@
+ 
+- **Customization:** Extensions enable you to customize PostgreSQL to suit your specific requirements. You can add new data types, functions, or operators that are not available in the core PostgreSQL distribution. This flexibility allows you to extend the capabilities of the database to handle specialized data or perform specific operations. 
+ 
+- **Simplified Installation:** Extensions provide a standardized installation process. They package all the necessary files, including the control file and installation scripts, making it easy to install and manage the extension within a PostgreSQL database. This simplifies the deployment of additional functionality and reduces the chances of errors during installation. 
+ 
+- **Version Compatibility:** Extensions are designed to be compatible across different versions of PostgreSQL. They abstract the implementation details and provide a consistent interface, allowing extensions to work seamlessly across compatible versions. This makes it easier to upgrade or migrate your database while retaining the functionality provided by the extensions. 
+ 
+- **Community Contributions:** PostgreSQL has a vibrant and active community that develops and maintains a wide range of extensions. These extensions cover various domains such as GIS, full-text search, JSON processing, time-series data, and more. By leveraging existing extensions, developers can benefit from community expertise and avoid reinventing the wheel. 
+ 
+- **Performance Optimization:** Extensions can also be used to optimize the performance of your PostgreSQL database. They can provide advanced indexing techniques, caching mechanisms, or query optimization algorithms that improve query execution speed and overall database performance.  
+ 
+Overall, PostgreSQL extensions enhance the flexibility, customization, and extensibility of the PostgreSQL database system. They allow developers and users to tailor the database to their specific needs, leverage community-contributed functionality, and simplify the installation and management of additional features.  
+# Basic Architecture:  
+The extension architecture follows a modular approach, where each extension is packaged as a separate unit. Let's explore the key components and how they work together: 
+ 
+1. **Extension Control File:** 
+    - The extension control file, typically named `extension_name.control`, is a plain text file that provides metadata and instructions about the extension. 
+    - It contains information such as the extension name, version, description, dependencies, installation script, and configuration options. 
+    - This file is used by PostgreSQL to manage the extension's installation, upgrade, and removal. 
```

Comment:
Maybe link to [the documentation](https://www.postgresql.org/docs/15/extend-extensions.html) for the control file.

Reply:
This is a doubt I had, so basically, Postgres is updated to newer versions every time. So I don't want my links to point to a previous version of postgres Doc, as there is a chance even in these 2 remaining months for the docs to change.

So I will edit all the links at the end of the project cycle.
But it is a good idea to comment something so I'll remember where the links are to be used